README.md

```diff
@@ -1,6 +1,6 @@
 # Project name
```

Comment:
While you are here, you might also want to set the project name and URL

Reply:
Sure

Reply:
Sure!

Reply:
I've updated the project name.
What would you like to set the URL as? Like is the project hosted anywhere?

Reply:
The URL should be https://rezultatevot.ro

Reply:
Okay